other/management/commands/push-notify.py

```diff
@@ -114,6 +115,11 @@ def get_rich_notification(notification):
             title = actor.title
             notification_extra = actor.link
 
+        # Comment
+        if isinstance(actor, Comment):
+            title = actor.text
+            notification_extra = actor.commented_by.user
```

Comment:
1) Title is limited to a very small number of characters, so it will inevitably get truncated.
2) Instead, change the title to the text of the **complaint**, since that is what matters. If I get a notification, I should immediately know *what* this notification is about. Change the verb to a static "New comment on a complaint you're following", and put the text of the comment in `notification_large_content`.
3) It doesn't matter who has posted the comment, so no need to pass that in the notification. Instead, pass the id of the complaint as the extra so we can implement follow-able notifications.